### DISCUSSION TITLE
Block IPv4 or IPv6 records

### DISCUSSION THREAD

**Discussion by aer0nix** [DNSCrypt/dnscrypt-proxy#1975] [Ideas]
Hi. I use IPv6 Tunnel with 100 Mbps, but my ISP have Google Global Cache on local network with 1 Gbps. So, I need to block IPv6 records for some Google's domains for increase performance. Can you make feature for block several domains by list?

For example:
```

*.googlevideo.com      IPv4-only
*.gvt1.com      IPv4-only
...
```

**Comment by Kylvan-8** [comment]
Hello,

https://github.com/DNSCrypt/dnscrypt-proxy/wiki/Caching

Does that answer to your need ? Look at filter also.

o/

**Comment by aer0nix** [comment]
No. I didn't want to block all IPv6. I partly solve my problem with Firefox's 'network.dns.ipv4OnlyDomains' string, but this way well nice only for browser.
